.github/workflows/cargo-deny.yml

```diff
@@ -0,0 +1,42 @@
+name: cargo deny checks
+
+on:
+  pull_request:
+    paths:
+      - 'Cargo.lock'
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  cargo-deny:
+    strategy:
+      matrix: >-
+        ${{
+          fromJSON(
+            github.event_name == 'schedule'
+              && '{"ref":["main","release","release-proxy","release-compute"]}'
+              || format('{{"ref":["{0}"]}}', github.sha)
+          )
+        }}
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ matrix.ref }}
+
+      - name: Check rust licenses/bans/advisories/sources
+        run: cargo deny check --hide-inclusion-graph
```

Comment:
Along with checking for open vulnerabilities, `cargo deny` also checks licenses and some other things (probably).
We want to make only vulnerabilities check a scheduled one (as it doesn't depend on external data), and do the rest of checks in PRs (we probably should do a vulnerabilities check if `Cargo.toml` changed to not to add vulnerable dependencies)

Reply:
adjusted so that the scheduled runs only check advisories